output/components.sasslint.txt

```diff
@@ -7,8 +7,8 @@ src/Actions/ActionFile/ActionFile.scss
   5:12  error  Strings must use single quotes  quotes
 
 src/FilterBar/FilterBar.scss
-   3:34  warning  Hex notation should all be upper case  hex-notation
-  28:6   warning  Vendor prefixes should not be used     no-vendor-prefixes
+   3:33  warning  Hex notation should all be upper case  hex-notation
```

Comment:
fix lint

Reply:
we said prettier is doing that lower case, so rule in conflict.